### ISSUE TITLE
Enhance a link to Help

### ISSUE THREAD

**Issue by julia-vaseva** [Microsoft/vso-intellij#97]
The link on TFVC Setting page and in "Checkout from Team Services" dialog should lead to http://java.visualstudio.com/Docs/tools/intelliJ

![2017-08-09_1810](https://user-images.githubusercontent.com/30112159/29129008-28046020-7d2e-11e7-9f76-2b3abdedb0e2.png)
![2017-08-09_1813](https://user-images.githubusercontent.com/30112159/29129140-7d283342-7d2e-11e7-9afd-cd48d87edbcb.png)

**Comment by leantk** [comment]
@julia-vaseva I've been looking into this and to me it looks like the Help option can only be directed to a www.jetbrains.com sub-site. I'll check with some of the engineers to see if that's true.

Thanks,
-Leah

**Comment by dmimat** [comment]
@leantk @julia-vaseva That's true, currently there is no way to use custom URLs in the plugins' context help. Here is the issue: https://youtrack.jetbrains.com/issue/IDEA-166122. Please feel free to watch and upvote.

**Comment by dmimat** [comment]
@leantk, some good news. Turns out that it is possible to add custom URLs for plugins' context help by implementing `com.intellij.openapi.help.WebHelpProvider` in the plugin's code.
